### DISCUSSION TITLE
Saving the cosmograph as an image?

### DISCUSSION THREAD

**Discussion by ymerouani** [cosmograph-org/cosmos#65] [Q&A] [answered]
What an amazing tool you have made! Effortless, easy to use and beautiful!
Of course, I would love to download the image of the network but I cannot find any option for that. Does anyone have any ideas on how to go about this except for taking screenshots?

**Comment by Javingka** [comment, marked as answer]
Hi @matriim 

Try this:

In your html code, include a button you will use to take the screenshot and download the image
```html
    <div  id="download">Download Image</div>
```

and in your javascript code create a function to manage the download whenever the button is clicked.

```javascript
var downloadButton = document.getElementById("download");
function download() {
    let canvas = document.getElementById('your-canvas-id'); // replace 'your-canvas-id' with the id of your canvas
    let screenshot = canvas.toDataURL("image/png");
    let a = document.createElement('a');
    a.href = screenshot;
    a.download = 'screenshot.png';
    a.click();
}
downloadButton.addEventListener("click", download);
```

This will do the trick.

In [this](https://codesandbox.io/s/cosmos-usage-example-forked-fzyztx?file=/src/index.ts) sandbox I tested.

cheers

**Comment by rokotyan** [comment]
@Javingka Thanks for posting an answer, that should do the job!

@matriim And if you're using the Cosmograph App on our website, you can press `Shift + S` to save your graph as an image.
